### ISSUE TITLE
Project refreshes on save, but doesn't compile LESS-file

### ISSUE THREAD

**Issue by raysreclame** [bdkjones/CodeKit2#627]
Hi there,

First of all: i love your app. But after updating to macos sierra my less-files won't auto-compile on save anymore. Tried trashing the project and adding it again, but no luck on that. I have updatet to Codekit 2.9, but that didn't help either... Now i'm hitting 'compile this file' manually and that works, but that really ruins my day.

Greetings, Raymond

**Comment by bdkjones** [comment]
Hi Ray. I haven't had a report of this from anyone else, so there's something specific to your Mac happening. Have you tried rebooting, as a start?

**Comment by bdkjones** [comment]
Also: which editor are you using to save your Less files? Have you tried saving them from a different editor?

**Comment by raysreclame** [comment]
Hi Bryan, that’s what i thought too, but it stopped working on both my iMac and my MacbookAir.
Restarted them multiple times.

**Comment by bdkjones** [comment]
Ok, which editor are you using to save the files?

**Comment by raysreclame** [comment]
I use Coda 2.5 from Panic.

> Op 27 sep. 2016, om 18:28 heeft Bryan Jones notifications@github.com het volgende geschreven:
> 
> Ok, which editor are you using to save the files?
>

**Comment by bdkjones** [comment]
Try saving the file in a different editor and let me know what happens. Also: is the project on your local disk, or on a removable one or NAS drive?

**Comment by raysreclame** [comment]
Oh darn, that did the trick. Used another editor and the file gets compiled. For your reference, i do work on a NAS drive. Guess i should be contacting someone at Panic. Thank you for your help and time!

> Op 27 sep. 2016, om 18:40 heeft Bryan Jones notifications@github.com het volgende geschreven:
> 
> Try saving the file in a different editor and let me know what happens. Also: is the project on your local disk, or on a removable one or NAS drive?
>

**Comment by schoien** [comment]
I have the exact same problem, did you solve it?

**Comment by raysreclame** [comment]
No, bot able to solve it in combination with Coda. Another editor works fine though. If Coda won't update/ fix it i might consider another editor.

**Comment by schoien** [comment]
Working for me as well with another editor ... But kind of rough path to switch editor ... Have you reached Panic support? hard to find them, seems twitter is only option

**Comment by raysreclame** [comment]
I know it's a rough path, but Coda (trough Twitter) didn't or couldn't really help. They state they didn't hear this issue from anybody else. Meanwhile i'm losing time and money, so another editor might get me back in track.

**Comment by davipt** [comment]
@raefconcepts are you by any chance running macOS Sierra beta 10.12.1 beta (16B2338c)?

I've noticed the same behavior everywhere else. Saving a file on Eclipse does not trigger the checkstyle check. Saving a file somewhere else, Eclipse or anything else, does not update SourceTree, and many other cases. My gut feeling is that something got broken on the filesystem change notification on this beta. This was not happening on the previous version.

**Comment by schoien** [comment]
Im using MacOs Sierra, but no beta. Seems the problem only occurs with projects located on network storage (AFP or SMB), not local drive.

**Comment by raysreclame** [comment]
@davipt I'm also not on beta, but on the release version of macOS Sierra. If that would be the bug, it would be strange that saving with another editor shows no issue at all.
